spring-backend/src/main/kotlin/com/example/sztoswro/member/MemberDTO.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.sztoswro.member
+
+data class MemberDTO(val id: Long,
+                     val name: String,
+                     val lastName: String,
+                     val email: String,
+                     val username: String,
+                     val password: String) {
```

Comment:
Nie wiem czy to tak działa, ale nie powinniśmy wysyłać hasła

Reply:
Z frontu w dto musi przyjść hasło, to jest komunikacja wewnątrz systemu. Potem w bazie jest trzymana zaszyfrowana postać.